### PR TITLE
Add a warning if user orders in a POD that is being closed

### PR DESCRIPTION
**Pull request by caberos**
https://github.com/softlayer/softlayer-python/issues/1587

### CODE REVIEW

**Comment by BrianSantivanez** [comment]
Running with virtual guests
image

Running with hardware
image

please add the warning to `place order`

After add  to `place order` and it passes all checks it is ok to me

File: SoftLayer/CLI/order/place.py [hunk 1]
```diff
@@ -90,6 +95,10 @@ def cli(env, package_keyname, location, preset, verify, billing, complex_type,
             ])
 
     else:
+        print(args)
```
Comment (allmightyspiff) [inline SoftLayer/CLI/order/place.py:98]: Looks like this should be removed

**Review by allmightyspiff — APPROVED** [review]
Code looks good, just need to fix the unit tests. Thanks
And remove that one errant print(args) line

**Review by allmightyspiff — APPROVED** [review]

**Comment by allmightyspiff** [comment]
@jonlowe-ibm I've got an issue open about it  https://github.com/softlayer/softlayer-python/issues/1687
Will be fixed in the next version. For now they can just ignore the warning if it doesn't have a datacenter listed in it.

Thanks for letting us know

**Pull request merged**